contracts/ReflectionToken/ReflectionToken.sol

```diff
@@ -10,6 +10,9 @@ import "@openzeppelin/contracts/access/Ownable.sol";
 import "@openzeppelin/contracts/utils/Address.sol";
 
 contract ReflectionToken is IReflectionToken, Ownable {
+    // Review: 
+    // - TODO: Should comment that fee is the numerator in case the denominator is maxFee.
+    // - TODO: can you separate the Fee struct to another contract having itnernal methods?
```

Comment:
Well, I don't think there is a problem with current struct

Reply:
It's no problem, but very ugly if you combine every level of methods into one single contract.
I propose to keep one contract including one or two main data structures, and methods to read and write their data structure.
I know it takes a time to fix, so priority is medium.

Reply:
Sorry but you don't seem to know about Critical, Hight, Medium, Low, Information levels.

Reply:
Sorry, you don't know even the basic knowledge of OOP....

Reply:
Anyway, I'm focusing on the interface and operation as of now. The quality of code will be managed later on.